content/includes/nginx-plus/oss-plus-comparison.md

```diff
@@ -0,0 +1,5 @@
+---
+docs:
+---
+
+{{< call-out "note" >}}For a detailed comparison between NGINX Plus and NGINX Open Source, refer to [Differences between NGINX OSS and NGINX Plus](https://www.f5.com/placeholder) on the F5 website.{{</ call-out >}}
```

Comment:
```suggestion
{{< call-out "note" >}}For a detailed comparison between NGINX Plus and NGINX Open Source, refer to [Differences between NGINX Open Source and NGINX Plus](https://www.f5.com/placeholder) on the F5 website.{{</ call-out >}}
```

Reply:
I’m not sure what the title of the Marketing doc is, but in tech docs we’ve agreed to refer to NOSS as NGINX Open Source so our global audience knows what we mean.